src/components/hr/hr.component.js

```diff
@@ -26,9 +36,12 @@ Hr.propTypes = {
   /** Margin bottom, this value will be multiplied by the theme spacing constant (8) */
   mb: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 7]),
   /** Margin left, any valid css value */
-  ml: PropTypes.string,
+  ml: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   /** Margin right, any valid css value */
-  mr: PropTypes.string
+  mr: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  /** Breakpoint for adaptive left and right margins (below the breakpoint they go to 0).
+   * Enables the adaptive behaviour when set */
+  adaptivemxBreakpoint: PropTypes.number
```

Comment:
Should we change this?
```suggestion
  adaptiveMxBreakpoint: PropTypes.number
```